bcda/api/v1/api.go

```diff
@@ -285,6 +293,28 @@ func ServeData(w http.ResponseWriter, r *http.Request) {
 	} else {
 		http.ServeFile(w, r, fmt.Sprintf("%s/%s/%s", dataDir, jobID, fileName))
 	}
+
+	// Logging request for auditing
+	var jobIdInt, err = strconv.ParseUint(jobID, 10, 64)
+
+	if err == nil {
+		var jobKey, err = repository.GetJobKey(context.Background(), uint(jobIdInt), strings.TrimSpace(fileName))
+
+		if err == nil {
+			var authData, ok = r.Context().Value(auth.AuthDataContextKey).(auth.AuthData)
+
+			if ok {
+				log.API.Infof("ACO_ID: '%s', CMS_ID: '%s', Resource: '%s'", authData.ACOID, authData.CMSID, jobKey.ResourceType)
```

Comment:
I think we only need this logging statement; the 3 else blocks below indicate other issues with the request and or the job in question that would result in a non 200 response from ServeData.